src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestV4ServiceImplTest.java

```diff
@@ -0,0 +1,764 @@
+    private long mockCurrentTimeMillis;
+
+    @BeforeEach
+    void setUp() throws Exception {
+        mockAccessToken = buildMockAccessTokenFromUUID();
+        mockConcurEndpoint = new MockConcurRequestV4ServiceEndpoint(mockAccessToken,
+                RequestV4DetailFixture.PENDING_EXTERNAL_VALIDATION_TEST_REQUEST_JOHN_TEST,
+                RequestV4DetailFixture.PENDING_APPROVAL_TEST_REQUEST_JOHN_TEST,
+                RequestV4DetailFixture.PENDING_EXTERNAL_VALIDATION_REGULAR_REQUEST_JOHN_DOE,
+                RequestV4DetailFixture.CANCELED_TEST_REQUEST_JOHN_TEST,
+                RequestV4DetailFixture.APPROVED_TEST_REQUEST_JANE_DOE,
+                RequestV4DetailFixture.PENDING_EXTERNAL_VALIDATION_TEST_REQUEST_JANE_DOE,
+                RequestV4DetailFixture.PENDING_EXTERNAL_VALIDATION_INVALID_TEST_REQUEST_JANE_DOE,
+                RequestV4DetailFixture.SENTBACK_INVALID_TEST_REQUEST_JANE_DOE,
+                RequestV4DetailFixture.PENDING_COST_APPROVAL_REGULAR_REQUEST_BOB_SMITH,
+                RequestV4DetailFixture.PENDING_EXTERNAL_VALIDATION_INVALID_REGULAR_REQUEST_BOB_SMITH,
+                RequestV4DetailFixture.PENDING_EXTERNAL_VALIDATION_REGULAR_REQUEST_BOB_SMITH,
+                RequestV4DetailFixture.NOT_SUBMITTED_REGULAR_REQUEST_BOB_SMITH);
+        
+        mockHttpServer = new MockRemoteServerExtension();
+        mockHttpServer.initialize(mockConcurEndpoint);
+        mockConcurBackendServer = mockConcurEndpoint.getMockBackendServer();
+        baseRequestV4Url = mockConcurEndpoint.getBaseRequestV4Url();
+        
+        concurParameters = buildTestParameters(baseRequestV4Url);
+        concurProperties = buildTestProperties();
+        
+        ConcurBatchUtilityService mockConcurBatchUtilityService = buildMockConcurBatchUtilityService();
+        testDateTimeService = buildDateTimeService();
+        easternTimeZone = DateTimeZone.forTimeZone(TimeZone.getTimeZone(US_EASTERN_TIME_ZONE));
+        mockCurrentTimeMillis = getTimeInMilliseconds(DEFAULT_MOCK_CURRENT_DATE);
+        
+        requestV4Service = new TestConcurRequestV4ServiceImpl();
+        requestV4Service.setSimulateProductionMode(false);
+        requestV4Service.setConcurBatchUtilityService(mockConcurBatchUtilityService);
+        requestV4Service.setConcurEventNotificationV2WebserviceService(
+                buildConcurEventNotificationV2WebserviceService(mockConcurBatchUtilityService));
+        requestV4Service.setConfigurationService(buildMockConfigurationService());
+        requestV4Service.setDateTimeService(buildSpiedDateTimeService(testDateTimeService));
+        requestV4Service.setConcurAccountValidationService(buildMockConcurAccountValidationService());
+    }
+
+    @SuppressWarnings("deprecation")
+    @AfterEach
+    void tearDown() throws Exception {
+        IOUtils.closeQuietly(mockConcurEndpoint);
+        IOUtils.closeQuietly(mockHttpServer);
+        IOUtils.closeQuietly(mockConcurBackendServer);
```

Comment:
A few of these test classes don't have resources or other handlers being left open, so I could potentially exclude their close() logic and let the Java GC clean them up instead. However, the "mockHttpServer" actively runs a local HTTP server, so that will still need to be closed or shut down accordingly. I may rename that variable to keep the intent a little clearer.